Secure-Software-Development-Policy/Zowe-SSDP-SDLC/Best-Practices/Test/quality.md

```diff
@@ -0,0 +1,27 @@
+# Quality
+
+The following list is based on the [Core Infrastructure Initiative](https://bestpractices.coreinfrastructure.org/en/criteria/0)
+
+## Working build system
+
+* The squad uses a working build system that can automatically rebuild the software from source code. 
+* It is SUGGESTED that Zowe Jenkins or Github Actions are used for building the software.
```

Comment:
Do we say anything about build system access control and security? Important since we don't want builds to be compromised.